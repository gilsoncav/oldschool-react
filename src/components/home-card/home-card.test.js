import HomeCard from './home-card';

describe('HomeCard component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<HomeCard>Test</HomeCard>);
    expect(wrapper).toMatchSnapshot();
  });
});
