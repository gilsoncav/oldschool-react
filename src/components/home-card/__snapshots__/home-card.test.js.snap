// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeCard component renders correctly 1`] = `
.c0 {
  padding: 20px;
  background-color: #373e49;
  border-radius: 15px;
}

.c0 button {
  float: left;
  margin-left: 5px;
  margin-top: 5px;
}

.c0 img {
  float: right;
  margin-right: 15px;
  margin-top: 15px;
  height: 25px;
}

.c0 > div:nth-child(3) {
  padding: 60px 0;
}

<div
  className="c0"
  id="home-card"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <button
      className="ant-btn btn-secondary ant-btn-lg"
      onClick={[Function]}
      type="button"
    >
      <i
        aria-label="icon: rollback"
        className="anticon anticon-rollback"
      >
        <svg
          aria-hidden="true"
          className=""
          data-icon="rollback"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M793 242H366v-74c0-6.7-7.7-10.4-12.9-6.3l-142 112a8 8 0 0 0 0 12.6l142 112c5.2 4.1 12.9.4 12.9-6.3v-74h415v470H175c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h618c35.3 0 64-28.7 64-64V306c0-35.3-28.7-64-64-64z"
          />
        </svg>
      </i>
      <span>
        Back
      </span>
    </button>
  </a>
  <img
    alt="Logo"
    src="logo.png"
  />
  <div>
    Test
  </div>
</div>
`;
